addons/web/static/src/views/graph/graph_renderer.js

```diff
@@ -379,10 +381,10 @@ export class GraphRenderer extends Component {
         const data = this.model.data;
         // style/complete data
         // give same color to same groups from different origins
-        const colors = data.labels.map((_, index) => getColor(index));
+        const colors = data.labels.map((_, index) => getColor(index, this.cookies.current.color_scheme));
         for (const dataset of data.datasets) {
             dataset.backgroundColor = colors;
-            dataset.borderColor = BORDER_WHITE;
+            dataset.borderColor = getBorderWhite(this.cookies.current.color_scheme);
```

Comment:
Could we extract this call to a function's constant to avoid unnecessary calls to the `getBorderWhite` function and the cookie's value ?